src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/GeneratedJavadocTokenTypesTest.java

```diff
@@ -220,7 +220,7 @@ public void testRuleNumbers() {
         assertEquals(MSG, 63, JavadocParser.RULE_linkTag);
         assertEquals(MSG, 64, JavadocParser.RULE_metaTag);
         assertEquals(MSG, 65, JavadocParser.RULE_paramTag);
-        assertEquals(MSG, 66, JavadocParser.RULE_wrongSinletonTag);
+        assertEquals(MSG, 66, JavadocParser.RULE_wrongSingletonTag);
```

Comment:
We might not be able to fix this as I assume the rule is part of the tree.

Reply:
We can can change a name, but we can not make it as minor spelling commit, we need to make issue on this and probably mark it as braking compatibility

Reply:
http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html

We do not change any of this name. So users should not be affected. Changed name is internal.